backend/scripts/diagnose_firecrawl.py

```
@@ -0,0 +1,268 @@
+#!/usr/bin/env python3
+"""
+Firecrawl 连接诊断脚本
+用于测试 Firecrawl API 连接、代理配置和网络问题
+"""
+
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)